src/nutrients_plankton_bacteria_detritus.jl

```diff
@@ -55,12 +55,12 @@ Biogeochemical functions
 """
 Base.@kwdef struct NutrientsPlanktonBacteriaDetritus{FT} <: AbstractBiogeochemistry
     maximum_plankton_growth_rate :: FT = 1/day
-    maximum_bacteria_growth_rate :: FT = 0.5/day
+    maximum_bacteria_growth_rate :: FT = 1/day
     bacteria_yield :: FT               = 0.2
     quadratic_mortality_rate :: FT     = 1/day # m³/mmol/day
     nutrient_half_saturation :: FT     = 0.1   # mmol m⁻³
     detritus_half_saturation :: FT     = 0.1   # mmol m⁻³
-    PAR_half_saturation :: FT          = 10.0  # W m⁻²
+    PAR_half_saturation :: FT          = 10/700.0  # nondim (from avg surface PAR of 700 W m⁻²)
```

Comment:
@emilyzakem if you mark this PR as ready for review then I can approve and we can merge